src/features/game/events/landExpansion/plant.ts

```diff
@@ -323,6 +323,44 @@ export function getCropYieldAmount({
     }
   }
 
+  const isAdvancedLevelCrop =
```

Comment:
For long variations of buffs it isn't always necessary to include them on the front-end - This is since the backend will apply these buffs onto the fields before they are ever harvested.

We can keep them here, but in future cases don't feel the need to always include them 👍

Reply:
> For long variations of buffs it isn't always necessary to include them on the front-end

Recommendation:
I think it's good to include them in the front-end for two reasons:
(a) clarity to players on how buffs behave (for example, to this day we don't know for sure if Rock Golem applies x3 or +2)
(b) catch bugs earlier, for example Sir Goldensnout's buff being applied "too early" (an add before the muls)

@adamhannigan